client/src/components/Announcements.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import Announcement from './Announcement'
+import './css/Announcements.css'
+
+class Announcements extends Component {
+
+    constructor(props) {
+      super(props);
+      
+      this.state = {
+        announcements : this.props.announcements,
+          id: this.props.id
+      }
+    }
+
+    //TODO: Add KEY FOR ANNOUNCEMENT
+    render() {
+        return(
+            <div>
+                <br></br>
+                
+                <ul id="Announcements">
+                <h2>Announcements</h2>
+                    {this.state.announcements.map(function(d, id){
```

Comment:
It would probably be a good idea to throw this into a function, that way it'll be easier to read and debug later on.